.github/workflows/protocol.yml

```diff
@@ -65,7 +65,12 @@ jobs:
 
       - name: protocol - Deploy L1 Contracts
         working-directory: ./packages/protocol
-        run: pnpm deploy:hardhat
+        run: |
+          anvil &
+          while ! nc -z localhost 8545; do
+            sleep 1
+          done
+          pnpm deploy:foundry
```

Comment:
will this machine have anvil and foundry? we might need to install foundry on this machine. we might need to curl the foundry binaries `curl -L https://foundry.paradigm.xyz | bash` && `foundryup`